.github/workflows/test.yml

```
@@ -21,14 +21,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [windows-latest]
 
     steps:
     - uses: actions/checkout@v2
+    - run: |
+        npm install
+        npm run all
     - uses: ./
+    - name: Run conda list
+      run: conda list
+    - name: init pwsh
+      run: conda init powershell
```

Can't hurt to init conda for all profiles - we do that by default:
```suggestion
      run: conda init --system powershell
```

AFAICT the `--system` flag is still completely undocumented  :(
https://github.com/conda/conda/blob/5cb5ac11c900e8ad39936c1e31c5fa1ac80d74ac/conda/cli/main_init.py#L44